arch/lkl/kernel/signal.c

```diff
@@ -38,6 +38,17 @@ static void handle_signal(struct ksignal *ksig, struct ucontext *uctx)
     ksig->ka.sa.sa_handler(ksig->sig, (void*)&ksig->info, (void*)uctx);
 }
 
+/*
+    While you might think this should call move_signals_to_task, send_current_signals
```

Comment:
Please explain what this does and why, not why I might think it would do something different.

Reply:
done